app/components/donation/donation-container.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const {
+  Component,
+  computed: { bool }
+} = Ember;
+
+export default Component.extend({
+  classNames: ['donation-container'],
+  donationAmount: 0,
+  projectTitle: null,
+
+  canDonate: bool('projectTitle')
```

Comment:
I should also note that there shouldn't be a case of having no `projectTitle` but I would like this component to be as stateless as possible in regards to where it is used.